sotodlib/io/load_smurf.py

```diff
@@ -927,6 +933,20 @@ def index_metadata(self, min_ctime=16000*1e5, max_ctime=None, stop_at_error=Fals
             
         session.close()
 
+    def lookup_file(self, filename, fail_ok=False):
+        """Lookup a file's observations details in database. Meant to look
+        and act like core.metadata.obsfiledb.lookup_file
+        """
+        session = self.Session()
+        file = session.query(Files).filter(Files.name==filename).one_or_none()
+
+        if file is None and fail_ok:
+            logger.debug(f"Did not find file {filename} in database")
+            return None
+
+        return { 'obs_id': file.obs_id,
+                 'detsets' : file.detset,
```

Comment:
Yeah I should've circled back once I found the suprsync bug, since I'm almost sure this is 100% related to the tunefile syncing problem. This should definitely be ignored, since the database has a None value, because of our sync issues.